server/app.py

```
@@ -77,68 +84,150 @@ async def lifespan(app: FastAPI):
-                "[WARN] pending registration 3회 실패함. DE 계속 실행, "
-                "수동 fallback 의존."
-            )
-        # 분기 2 내부 raise 금지 — yield까지 정상 진행
-        app.state.pending_registered = pending_registered
+            else:
+                # 분기 3: SEQUENTIAL (backward-compat)
+                await register_to_backend(public_url, settings.engine_secret_key)
+                app.state.heartbeat_task = asyncio.create_task(
+                    start_heartbeat(public_url, settings.engine_secret_key)
+                )
+        except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)